docs/api/remote-procedure-calls-blockchain.md

```diff
@@ -1409,7 +1396,7 @@ TXID | string (hex) | Required<br>(exactly 1) | The TXID of a transaction in the
 Name | Type | Presence | Description
 --- | --- | --- | ---
 `result` | object | Required<br>(exactly 1) | A object containing transactions currently in the memory pool.  May be empty
-→<br>`size` | number (int) | Required<br>(exactly 1) | The size of the serialized transaction in bytes
+→<br>`vsize` | number (int) | Required<br>(exactly 1) | The size of the serialized transaction in bytes
```

Comment:
The description needs to be updated to match the help output as well.
```suggestion
→<br>`vsize` | number (int) | Required<br>(exactly 1) | The virtual transaction size. This can be different from actual serialized size for high-sigop transactions.
```